default/stringtables/en.txt

```diff
@@ -6681,10 +6681,20 @@ QUICK_START_GUIDE_TITLE
 **Quick Start Guide**
 
 QUICK_START_GUIDE_TEXT
-'''This is a brief introduction to FreeOrion gameplay. A somewhat less brief Quick Play Guide is available at freeorion.org (current URL: <url  http://freeorion.org/index.php/V0.4_Quick_Play_Guide>freeorion.org/index.php/V0.4_Quick_Play_Guide</url> ).
+'''This Pedia provides information about many aspects of FreeOrion, including mechanics and content, as well as in-game objects and empires during a game. The pedia can be accessed from the top level index page, which provides categorized lists of articles, by entering text in the box at the top of the pedia page to search articles for the entered text, or via right-click popup menu links from many in-game objects in the interface.
 
-[[DETAILED_PEDIA_REFERENCE]]
-Important sections include [[encyclopedia CATEGORY_GAME_CONCEPTS]], [[encyclopedia CATEGORY_GUIDES]], and the [[encyclopedia ENC_METER_TYPE]] list.
+At the start of a new game, there are three important issues that should be addressed:
+
+1. Fleets are initially idle at homeworld's planetary system. Visible in the [[encyclopedia MAP_WINDOW_ARTICLE_TITLE]] is a star map, with the home system selected and highlighted in the center. Next to it, a bit up and to the right, is the fleets icon. Fleets can be selected by left-clicking on the fleet symbol on the map, which will open the fleets window, where all the fleets in a system and their contained ships can be viewed. If a fleet is selected, it can be given movement orders by right-clicking a nearby target star that is connected to the fleet's system by a starlane. Fleets that are not given move orders will stay idle. A good first step is often to send scout fleets to explore nearby systems, which will reveal more about their contents and any further starlane connections beyond them. See also [[encyclopedia FLEET_MOVEMENT_ARTICLE_TITLE]].
```

Comment:
```suggestion
1. Fleets are initially idle at homeworld's planetary system. Visible in the [[encyclopedia MAP_WINDOW_ARTICLE_TITLE]] is a star map, with the home system icon selected and highlighted in the center. At the top right of the system icon is a fleet icon. Fleets can be selected by left-clicking on the fleet icon on the map, which will open the fleets window, where all the fleets in a system and their contained ships can be viewed. If a fleet is selected, it can be given movement orders by right-clicking a nearby target star that is connected to the fleet's system by a starlane. Fleets that are not given move orders will stay idle. A good first step is often to send scout fleets to explore nearby systems, which will reveal more about their contents and any further starlane connections beyond them. See also [[encyclopedia FLEET_MOVEMENT_ARTICLE_TITLE]].
```